src/backend/distributed/utils/maintenanced.c

```diff
@@ -622,6 +622,13 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 			timeout = Min(timeout, deadlockTimeout);
 		}
 
+		/* Periodically persist the logical clock value */
+		if (!RecoveryInProgress())
+		{
+			InvalidateMetadataSystemCache();
```

Comment:
why are we invalidating metadata cache here ?

Reply:
This is coming from the Citus daemon
![image](https://user-images.githubusercontent.com/44680808/163492061-06cdf74d-658b-4ea2-8fae-c9a1272cf7c7.png)